src/Analyzer.Core.BuiltInRuleTests/Tests/TA-000003/TA-000003.json

```diff
@@ -9,6 +9,14 @@
       {
         "LineNumber": 40,
         "Description": "siteConfig.ftpsState set to an undesired value"
+      },
+      {
+        "LineNumber": 161,
```

Comment:
I think one resource that produces one error is enough validation that the nested template was processed